Source/IGListAdapter.m

```diff
@@ -759,6 +760,23 @@ - (__kindof UICollectionReusableView *)dequeueReusableSupplementaryViewOfKind:(N
     return [collectionView dequeueReusableSupplementaryViewOfKind:elementKind withReuseIdentifier:identifier forIndexPath:indexPath];
 }
 
+- (__kindof UICollectionReusableView *)dequeueReusableSupplementaryViewOfKind:(NSString *)elementKind
+                                                         forSectionController:(IGListSectionController <IGListSectionType> *)sectionController
+                                                                      nibName:(NSString *)nibName
+                                                                       bundle:(NSBundle *)bundle
+                                                                      atIndex:(NSInteger)index {
+    IGAssertMainThread();
```

Comment:
Sorry for that, I'm trying this framework, hope to have this feature ASAP, without attention to details